src/core/standard_fonts.js

```diff
@@ -793,9 +892,14 @@ const getSupplementalGlyphMapForCalibri = getLookupTableFactory(function (t) {
   t[1086] = 45;
 });
 
-function getStandardFontName(name) {
+function getStandardFontName(name, xfa = false) {
```

Comment:
Instead of of adding a param, can we make this a different function instead? It returns a different type for xfa fonts and they aren't really standard fonts. Maybe `getXfaFontInfo`?